pallets/ds-accounts/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = 'FRAME pallet template for defining custom runtime logic.'
 edition = '2018'
 homepage = 'https://substrate.dev'
 license = 'Apache-2.0'
-name = 'pallet-template'
+name = 'pallet-ds-accounts'
 repository = 'https://github.com/mixbytes/substrate-project-template'
```

Comment:
Just search for the word "template" in all project files using your favorite IDE..